lib/dictionary.js

```diff
@@ -10,9 +10,9 @@ var segmenter = new Segmenter(definitionLookup);
 
 function start(){
 	console.log("Hanzi is compiling dictionary...");
-	
+
 	//Reading in CCEDICT
-	var readFile = fs.readFileSync(__dirname + '/dicts/cedict_ts.u8', 'utf-8');
```

Comment:
Heya @zurawiki,

good question. Way back when I started this, it used tabs, but I've since moved to spaces. So I prefer that now. Which is why there's some discrepancy.

Some data files though are tab separated, so that should remain in place. Like the frequency files.